tests/test_views.py

```diff
@@ -844,13 +845,13 @@ def test_alias_usage_view(self):
                 str(self.page),
             ),
         )
-        self.assertRegex(
-            str(response.content),
-            r'href="{}"[\w+]?>{}<\/a>'.format(
-                re.escape(root_alias.get_absolute_url()),
-                str(alias),
-            ),
-        )
+        # self.assertRegex(
```

Comment:
Do you have the output of what this was before vs what it is now?

Reply:
@Aiky30 Well spotted! I forgot to uncomment this. 

1. Alias rendering is not affected, and the above logic, of course, **stays the same**. (Also, this test does not relate to this.)
2. The "View usage" view does provide links (for convenience) to objects that use the alias, using an object's potential `get_absolute_url` if it exists. For alias, this is removed with this PR.

What I planned to do (and did not do yet) is to provide a template tag that will get the appropriate URL for the object: `get_absolute_url` if it is a public object like a `PageContent`, or the preview endpoint if it is frontend-editable. If it is neither, then no link will be provided. 

Then you'll have links to `AliasContent` models or any other models that are frontend-editable but do not have independent URLs. Hold tight, I'll catch up on this quickly.

Thanks for pointing this out!

Reply:
One more thought: For frontend-editable models, it would IMHO make sense to always clink the preview endpoint for a consistent user experience.

Reply:
@Aiky30 Fixed 😎